packages/grid_client/scripts/blockchain.ts

```diff
@@ -82,13 +82,16 @@ async function main() {
   await grid3.disconnect();
 }
 
-async function createAccount(account_name: string, blockchain_type: blockchainType, ip?: string) {
+async function createAccount(account_name: string, blockchain_type: blockchainType) {
   try {
-    const account_created = await grid3.blockchain.create({
+    //change the relay to the desired network
+    const relay = "relay.dev.grid.tf";
+    const account: BlockchainCreateModel = {
```

Comment:
Please make sure to initialize the model in the main function and pass it as a parameter to the other function as you did in all the other scripts.

Reply:
should be fixed in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/8fc5cb9cad2588dbc9c9a3bb91da3c90db688e6b